src/Entity/UrlBase.php

```diff
@@ -0,0 +1,57 @@
+<?php
+
+namespace App\Entity;
+
+use Doctrine\ORM\Mapping as ORM;
+
+/**
+ * @ORM\Entity(repositoryClass="App\Repository\UrlBaseRepository")
+ */
+class UrlBase
```

Comment:
Ta nazwa jest trochę myląca. Wystalczyłoby samo Url lub jeszcze lepie UrlRedirect. W tym momencie masz UrlBase a wew jeszcze jedno urlBase ($baseUrl) i do tego $generatedUrl - to jest średnio intuicyjne